src/components/my-component/my-component.tsx

```diff
@@ -22,7 +22,7 @@ export class MyComponent {
    */
   @Prop() last: string;
 
-  private getText(): string {
```

Comment:
Thanks for the PR!

Just a note that this line was `public format(): string {` until three days ago when @manucorporat changed it for a commit called "chore(): add docs"

Unless there was a good reason for _that_ change, I would just change this line back to that, fix up the render and leave the test file be as-was.

No biggie either way, though, IMO.

Reply:
@kensodemann Thanks for the feedback, I guess we should wait for @manucorporat to decide?

Reply:
Discussed with @manucorporat. This issue stems from this change:

https://github.com/ionic-team/stencil-component-starter/commit/e3aca80cc55f69bc8c9c9d5f4146c6e5eef04b58#diff-34c2a6c626ee2612cd4f12b2c004a0b1

If you have a look there, the original `format()` logic was moved to a file called `util.ts` to encourage people to encapsulate logic like that in non-component classes or files (like you would use a "service" in Angular).

So the best solution to this is to create a `util.spec.ts` file and move the `format()` tests over there.

Two choices:

1. you can modify this PR to do that
1. this PR can be closed without merge and I will do that sometime this morning

Up to you. Either works well for me.

Reply:
@kensodemann 
And then what? do we leave the component without a test? I can Add a spec file for utils, but the component is still not tested, I guess this was meant to show how to test the stencil components, I guess most people already know how to test "regular" typescript/javascript.

Reply:
@liron-navon - have a look at `my-component.e2e.ts`. It is testing that the rendering occurs correctly.

The general philosophy being that components are mostly about rendering something and that rendered something is best tested via an `e2e` test whereas unit tests are for other logic, so components should have extensive `e2e` tests with minimal (or even non-existent) `unit` tests. The support classes / fils like `util.ts` should then have complete `unit` tests.

The unit test for a component could be as simple as:

```TypeScript
import { MyComponent } from './my-component';

describe('my-component', () => {
  it('builds', () => {
    expect(new MyComponent()).toBeTruthy();
  });
});
```

Reply:
I didn't see any updates to this so I went ahead and did what I stated above via a seperate commit.

Thank-you again for the PR and for bringing this to our attention. I am going to close this one as this was fixied with https://github.com/ionic-team/stencil-component-starter/commit/e65f5071df4658b725eddffb540b9ab7bf652803